metaci/build/models.py

```diff
@@ -534,26 +536,16 @@ def run_flow(self, project_config, org_config):
             raise FlowNotFoundError('Flow not found: {}'.format(self.flow))
         flow_config = FlowConfig(flow)
 
-        if settings.METACI_FLOW_SUBCLASS_ENABLED:
-            class_path = 'metaci.build.flows.MetaCIFlow'
-        else:
-            # Get the class to look up options
-            class_path = flow_config.config.get('class_path',
-                                                'cumulusci.core.flows.BaseFlow')
-
-        flow_class = import_class(class_path)
+        callbacks = None
```

Comment:
there aren't checks for if callbacks is none in teh flowrunner (though maybe there should be) so this needs to be a FlowCallback if its not a MetaCIFlowCallback

Reply:
There is:

        if not callbacks:
            callbacks = FlowCallback()

Reply:
oh. heh.